source/Cosmos.System2/Graphics/FullScreenCanvas.cs

```diff
@@ -35,7 +35,7 @@ private enum VideoDriver
             VGADriver
         }
 
-        static Canvas videoDriver = null;
+        public static Canvas videoDriver = null;
```

Comment:
Why do we need this to be public?

Reply:
so you can directly access framebuffer and many driver stuff
- CopyVRAM
- GetVRAM
- SetVRAM
- ClearVRAM
- Fill

Also i think that people should have direct access to grapic driver, only positives